### ISSUE TITLE
2015-06 Release

### ISSUE THREAD

**Issue by davelab6** [fontforge/fontforge#2323]
We skipped a May release, so I suggest we cut a new release for June ASAP :)

**Comment by larsenwork** [comment]
Yep, never heard back from @monkeyiq on https://github.com/fontforge/FontForge-Installer-Mac/issues/1 but I'd like for the new installer to be included in this release as it would allow the website instructions to be simplified as per my pr https://github.com/fontforge/fontforge.github.io/pull/85

**Comment by larsenwork** [comment]
Would be nice if #2307 #2312 were fixed on mac which they presumably are on windows/linux

**Comment by pathumego** [comment]
So what happens to assigned issues for 2015-05? move to 2015-06?

**Comment by larsenwork** [comment]
@davelab6 any plans for this

**Comment by davelab6** [comment]
@frank-trampe will you have time for a release this month, or should someone else make it? perhaps  @lpsandaruwan ? :)

**Comment by davelab6** [comment]
> So what happens to assigned issues for 2015-05? move to 2015-06?

I think you should set them to the milestones you think are realisic. With @adrientetar working on his defconqt and @frank-trampe also contributing much less recently, I think it would be better to prioritise the things you and @larsenwork think are most important so @lpsandaruwan can work on them. 

I believe @frank-trampe wants to entirely focus on stability fixes

**Comment by frank-trampe** [comment]
I'm still doing a lot of work on FontForge, but most of it has been on other people's pull requests. We can make a June release at any time, but we haven't accomplished much in the last few months.

**Comment by pathumego** [comment]
@larsenwork we have merged the new them right? So if we have that I think its ok to do a release. @frank-trampe there are enough improvements thats worth an update for a user :) I say we make a new release now. The next release could be a test release with new menu structure forJuly/ August.

**Comment by larsenwork** [comment]
I think it makes more sense to wait a bit - no reason to do a release just for the sake of it. 
@pathumego theme/UI updates etc. haven't been pr merged yet. #2245 could probably be merged now. 

I'd like to a a couple of more things to the mac installer and we're still waiting on figuring out where the installer should be merged in main repo/workflow: https://github.com/fontforge/FontForge-Installer-Mac/issues/1

And since we're (read: @lpsandaruwan) busy reorganising the menu I think it makes more sense to wait a bit. Especially since (correct me if I'm wrong) there haven't been any major bug fix / feature added.

**Comment by lpsandaruwan** [comment]
Final decision ?

**Comment by frank-trampe** [comment]
There's nothing final about waiting.

**Comment by lpsandaruwan** [comment]
Okay. :-)

**Comment by davelab6** [comment]
Errm when I look at https://github.com/fontforge/fontforge/commits/master I
see a loooooooot of commits starting "Fixes..." :) I think a release is
well overdue

**Comment by davelab6** [comment]
eg https://github.com/fontforge/fontforge/issues/2286 in itself would be
release worthy

**Comment by frank-trampe** [comment]
Okay. I'll start that ball rolling tomorrow.

**Comment by davelab6** [comment]
Wonderful! Thank you Frank!!

**Comment by pathumego** [comment]
@larsenwork can you take a look at the new UI announcement doc and comment? I think better to put it out there with the next release. 

https://docs.google.com/document/d/1bpShivntSBa8qkfD0agJdSb3dbDOhCXnNAF31FRRHzw/edit?usp=sharing

**Comment by larsenwork** [comment]
@pathumego as in august or this release?

**Comment by pathumego** [comment]
This release, So we are announcing that we are doing this. Anyway I think before completely switching to new menu structure, we could offer two builds with new and old menues for August release. And then switch completely and. Maintaining a branch with the old menu structure and offering instructions on how to build with old menu structure would be good.

**Comment by frank-trampe** [comment]
@pathumego, there's a good chance I'll be moving to a separate stability-oriented fork in the next month or so. If this happens, it may not be necessary to keep such a branch here.

**Comment by davelab6** [comment]
Maybe call the new UI "FontForge Nova"? Then we could get sane version
numbering again.

**Comment by frank-trampe** [comment]
So we would change the base package name to `fontforgenova`?

**Comment by davelab6** [comment]
You would keep the maintenance version on the mainline 'fontforge' so we
don't disrupt people who are using it, and then Andreas and Pathum work on
a 'fontforgesomething' and users can migrate to that when they wish.

**Comment by frank-trampe** [comment]
That seems like a good plan.

**Comment by larsenwork** [comment]
I'd rather we just keep one track and keep the menu in a branch until we've tested it thoroughly and then merge it into the main branch.
I think FontForge-something (why confuse people) is a bad idea and there's no hurry in getting the menu redesign out. 

We can make a test-build of the redesigned menu once we're "satisfied" to get user feedback.

I don't see the connection between name change and switching to a more sane version numbering.

**Comment by larsenwork** [comment]
The menu redesign doesn't change any functionality - it's just rearranging and making it prettier and more logical arranged.

**Comment by frank-trampe** [comment]
Our current version numbers are very large. 20150612 is going out soon. If the next version were 2.1.1 or something like that, it would be less than 20150612, and package managers would still identify 20150612 as the most recent version of the package named fontforge.

**Comment by larsenwork** [comment]
Hmm...there's no way to add a "date-stamp" for the package managers?

**Comment by larsenwork** [comment]
Or we could just start counting at 3.0.0 ?

**Comment by frank-trampe** [comment]
The date is irrelevant. (Sorry for using the word recent.) Sometimes (when a package has multiple releases, each with its own current minor version) 2.1.5 is newer than 3.0.0, but the package manager prefers the highest version unless instructed otherwise.

**Comment by frank-trampe** [comment]
3.0.0 is not bigger than 20150612.

**Comment by frank-trampe** [comment]
My FontForge time this year is limited, and, barring some unexpected event, I am unlikely to have enough time to evaluate and to improve large changesets to professional standards. So when the time comes to merge those, we'll still need to branch or to fork a stable version so that I can continue monotonic progress towards a stable product.

**Comment by larsenwork** [comment]
Ok, no other way to tell the pkg manager "behind the scene" that this is never? 

E.g. what the user sees is FontForge 2.0 and what the package manager sees is FontForge 2.000000000?

**Comment by davelab6** [comment]
The "nova" name is my suggestion for how to resolve the divergent
directions that the menu redesign and frank's maintenance efforts appear to
be going in.

Frank, will you accept the menu rearrangement in your stable version?

**Comment by larsenwork** [comment]
but why not combine those efforts in one package? How does rearranging the menus affect "core" stability? (asking because I don't know:)

**Comment by davelab6** [comment]
On 12 June 2015 at 17:14, Andreas Larsen <notifications@github.com> wrote:

> E.g. what the user sees is FontForge 2.0 and what the package manager sees
> is FontForge 2.000000000
>

Both those values are the same, and both are less than say 20,140,612.0
which would be the release number if we made a release today.

**Comment by frank-trampe** [comment]
That's not hypothetical. I'm making a release today.

**Comment by larsenwork** [comment]
> That's not hypothetical. I'm making a release today.

great, can you send me the mac app version before uploading so I can put it in the new installer?

**Comment by frank-trampe** [comment]
When I make the Macintosh binary, it's sort of a manual process and takes a while, so I may not be able to do that today. (Probably Monday.) I don't normally do it at all, but since our automatic packages seem to be stale, that seems a good thing for me to do.

**Comment by larsenwork** [comment]
@frank-trampe ok, I'm also trying to learn how to do it (and will happily do it for future releases) - I think I made an issue today with some of the problems but hasn't @lpsandaruwan figured out how to package for mac @pathumego ?

**Comment by larsenwork** [comment]
Here's how far I've gotten so far: https://github.com/fontforge/fontforge/issues/2356

**Comment by davelab6** [comment]
Please collaborate on learning how to do this, and if you get stuck then
lets see if we can get Ben Martin for an hour to help you out... he has a
shell script running on his mac which builds the app and uploads it to
his fuuko server, and he said that all the scripts he uses to do this are
in the main repo, so you should be good with the tools there, its just a
question of documenting them better so its easy for you to follow

On 12 June 2015 at 17:22, Andreas Larsen <notifications@github.com> wrote:

> @frank-trampe <https://github.com/frank-trampe> ok, I'm also trying to
> learn how to do it (and will happily do it for future releases) - I think I
> made an issue today with some of the problems but hasn't @lpsandaruwan
> <https://github.com/lpsandaruwan> figured out how to package for mac
> @pathumego <https://github.com/pathumego> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fontforge/fontforge/issues/2323#issuecomment-111621839>
> .
>



-- 
Cheers
Dave

**Comment by larsenwork** [comment]
> Both those values are the same, and both are less than say 20,140,612.0
> which would be the release number if we made a release today.

Ok - we use commas and full stop differently with numbers in Denmark so I always get the English wrong. 2.0 and 2.000000 were just an example - what I'm interested in knowing is if we can somehow keep a short number that we show people and a big number for the pkg manager? 

> Please collaborate on learning how to do this, and if you get stuck then
> lets see if we can get Ben Martin for an hour to help you out... he has a
> shell script running on his mac which builds the app and uploads it to
> his fuuko server, and he said that all the scripts he uses to do this are
> in the main repo, so you should be good with the tools there, its just a
> question of documenting them better so its easy for you to follow

Hence my issue earlier today :)

**Comment by davelab6** [comment]
We discussed this a while ago and concluded there's nothing we can do,
because while some package formats like RPM have 'epochs' for this
situation, many like homebrew do not :(

**Comment by larsenwork** [comment]
@davelab6 ok, what I wanted to now:) 
hmm...a suffix just reminds me of a tacky 90's car that has got a lousy makeover to me.

**Comment by davelab6** [comment]
MS is releasing a bunch of "Nova" families in Windows 10 ;)
https://en.wikipedia.org/wiki/List_of_typefaces_included_with_Microsoft_Windows

**Comment by ultrasquid** [comment]
If we have to rename the FontForge update for the sake of package managers,
perhaps "FontForgeNeue" would work, as "neue" seems to be a more popular
synonym for "new" when dealing with typefaces than "nova".

On Fri, Jun 12, 2015 at 3:11 PM, Dave Crossland <notifications@github.com>
wrote:

> MS is releasing a bunch of "Nova" families in Windows 10 ;)
>
> https://en.wikipedia.org/wiki/List_of_typefaces_included_with_Microsoft_Windows
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fontforge/fontforge/issues/2323#issuecomment-111630671>
> .
>



-- 
-- 
Jason Pagura
zimbach at gmail dot com

**Comment by davelab6** [comment]
Linguistically, Neue is German while Nova is Latin; but in anglophone
typography culture, Neue has an "old school" vibe, reminding of Linotype's
1983 "Helvetica Neue", while "Nova" has a much more contemporary feeling,
reminding of "Proxima Nova" which is currently the #1 best selling font on
http://www.myfonts.com/bestsellers/

So while 'neue' is more popular, that is because its an older trend, and I
have suggested 'nova' because it is trending today

**Comment by larsenwork** [comment]
How about FontForge 1.0S - cars use it, iPhone use it... :sunglasses: 

Or call it FontForge Nova in src + for pkg managers but strip the Nova in our build scripts?

**Comment by larsenwork** [comment]
If we absolutely have to change it for all users then "FontForge Nova" is just a really, really bad app name in my book. Suffixes in apps are only used (what I can think off) for crippled Light versions (e.g. Photoshop Express).
Then I'd rather call it something else entirely.

**Comment by frank-trampe** [comment]
Fontanova.

**Comment by davelab6** [comment]
Right. Collaboration is hard: either you guys agree on a way to work
together, or, you can softly fork the project together (my nova
suggestion), or, one of you can hard fork the project and go off and call
it something totally different (like FontAnvil and SortMill Tools already
did.)

I think as goals, this should be simple for users and for maintaining, so I
think the best thing is for Frank and you to be working on the same
codebase.

**Comment by jtanx** [comment]
I'm somewhat against the idea of a completely new fork - I can understand having a stable and beta branch though.

**Comment by larsenwork** [comment]
@davelab6 I'm not really following your talk about forking and codebases? What does it have to do with versioning/naming?

@frank-trampe hehe...the Casanova of font editors, could make for some interesting graphics/logo

**Comment by larsenwork** [comment]
@jtanx agreed, @lpsandaruwan is doing the code changes for the menu update but I don't see why we can't just do it in a branch and then thoroughly test it before merging back to master

**Comment by frank-trampe** [comment]
I think that forking is the best approach here. We have two entirely divergent sets of priorities without the resources necessary to pursue them in the same codebase (ensuring code quality in the face of large changesets).

I also think that a fork dedicated to stability and quality might be suitable for merging with Sorts Mill at some point in the future (since they exist for the same purpose), so the total number of projects might not increase in the long term.

**Comment by larsenwork** [comment]
@frank-trampe forgive my noob questions but aren't menu changes limited to a few set of files that are used just to draw the menus? And then the actual actions/functions are located in different files?

I guess what I can't see is how rearranging and cleaning up the menus affect stability / core functions of the app.

**Comment by larsenwork** [comment]
@pathumego is also very keen on adding new functionality to the menus but I'm strongly advocating on waiting to do so - so that step 1 in the menu rearranging is just that - a rearrangement.

**Comment by jtanx** [comment]
@frank-trampe  on an unrelated note to forking - I notice in the tarball you're including the full git version history as well as the `gnulib` folder. Since bootstrap has been run to generate the configure script, I don't think these files are necessary, and removing them would significantly cut back on the tarball size. At least, I don't think we should be including the git revision history in a tarball.

**Comment by frank-trampe** [comment]
Rearranging, adding, and removing widgets in FontForge (unfortunately) can be a complicated process. I'm also the person who inevitably ends up explaining why things are as they are and why even a bug-free change might not be a good change. These discussions take a lot of time. Depending upon the nature of the changes, the menu rework might not be a huge disaster in itself, but I don't expect this to be the only major change from the New FontForge Movement. (I sense a lot of energy.)

**Comment by frank-trampe** [comment]
@jtanx, we discussed the tarball size previously somewhere. The git folder is probably unnecessary most of the time (particularly since rpmbuild ought to get the version out of the spec file), but it didn't seem worthwhile to remove it. We include gnulib since some package build services deny network access except for installation of packages from standard repositories.

**Comment by jtanx** [comment]
But since the configure script should already exist, `bootstrap` shouldn't need to be called?

**Comment by larsenwork** [comment]
@frank-trampe I guess I sit somewhere between you guys because I think increased stability with the current functionality should be of highest priority together with making it easier to use (and the menu is currently not very intuitive for a beginner).

I think fancy new functionality is probably going to be a lot easier to do as side (web based?) projects instead of trying to learn this old lady new tricks.

**Comment by davelab6** [comment]
Jeremy, I agree about removing the `.git` directory, if someone needs that
they can prepare their own tarball; I also agree about not wanting a fork
and its better if we can work together.

Frank, I think menus are a special kind of widget in that their behavior is
tightly scoped and rearranging items should be a very simple process. The
menu re-organization is very important for users. They HATE that Element
menu with its jumble of items, it is a total mess. As Andreas mentions just
now, he and Pathum have taken your plea not to change things but just
reorder into account when designing the new menus, so it should not be a
big strain on you; and they are preparing documentation for the user
community to help them overcome any psychological change aversion in the
move to the better future.

Even if it is a big pull request in the end, I think it is important that
we change the menus to something that has been organized thoughtfully.

If you can successfully collaborate on this, then the next thing will be to
discuss what the next round of development should be about and a successful
collaboration will help everyone calibrate what is possible

**Comment by davelab6** [comment]
On 12 June 2015 at 20:19, Dave Crossland <dave@lab6.com> wrote:

> They HATE that Element menu with its jumble of items, it is a total mess


Also having different items in the same menus depending on the window type
is also an utter disaster

**Comment by larsenwork** [comment]
> Also having different items in the same menus depending on the window type
> is also an utter disaster

Yep...just like having duplicate menu items... :wink:

**Comment by frank-trampe** [comment]
We tried skipping bootstrap, but there are links (dependent upon the source location) in the config folder that bootstrap generates that need to be regenerated. There may be a way around this, but I cannot find the discussion.

There is also an option to bootstrap called `--gnulib-srcdir` for using an existing bootstrap directory.

**Comment by jtanx** [comment]
@frank-trampe okay, I can try looking into this, probably tomorrow.

**Comment by davelab6** [comment]
I think fundamentally Frank is assuming that FontForge's current state is
actually useful and today is a good point at which to start the stability
push.

But it is not clear to me that this is the case; in my experience, when
people try to use FontForge for real projects, they almost always give up
and switch to a proprietary editor, because it is too frustrating to use in
its current design. This is mostly due to the menu layout. There are also
serious problems with point selection using the mouse being inaccurate, it
crashes quite a lot, and some super basic features like line wrapping in
the metrics window are missing.

So I think it would be more wise to push ahead with the redesign of the
menus, add just a few other things based on the real world experience of
Andreas and Pathum who are actually using FontForge to make real fonts, and
then making a hard push for stability, starting with completing the
breakpad integration that monkeyiq started so that when users suffer a
crash then we actually get a backtrace from them.

**Comment by mskala** [comment]
Rearranging the menus sure would be a lot easier to do in GTK, Qt, or almost any GUI toolkit other than Gdraw.  As long as FontForge continues to use Gdraw, FontForge will be paying a punishingly high Gdraw tax on all GUI development.  And I hope any major GUI layout changes will be accompanied by getting rid of documentation and Web sites that describe or show screenshots of the old layout.  We were badly hurt by having the prominently featured subtitle of the main Web site be "Font editor, old version at (link to obsolete URL)" for years.  Google and other traffic generators pointed at the old Web site in preference, and didn't switch over until the old one finally went down.  We still get support questions from people who are reading the hopelessly obsolete non-English translated versions of the old documents.  The same will happen if there is a situation where "old GUI" documents and "new GUI" documents are online and easy to find at the same time, especially if the "new GUI" documents include links to the old ones.

Speaking of old Web sites, there's been some concern about other projects that have old versions on SourceForge, in that SourceForge has taken over the old accounts and posted their own packaged versions of the projects with malware added.  This seems a worrying possibility, especially for a project that is trying to move toward packaged binaries and unsophisticated users and has a name ending in "Forge" (which might suggest some connection to and legitimacy of SourceForge as a distribution site).  If a user gets malware or even just an obsolete non-working version from SourceForge, they will blame FontForge, not SourceForge.  What worries me most about the SourceForge malware scandal is that since it hit the news media, SourceForge management have in their public statements created the impression that they actually see nothing wrong with what they were doing and they intend to continue doing it in general, at most _maybe_ stopping in the case of individual projects when they get specific complaints.

Now, about forks.  If I marry a woman, and a few years later we find that it's not working out and we get divorced, okay, that kind of thing happens to a lot of people and maybe she just wasn't the right one for me.  If I then marry a second time and then immediately divorce again, okay, maybe she was a bad person too and that's not my fault either.  But by the time I'm on my third divorce, it's pretty likely that someone is going to point out, hey, what all three of those marriages had in common wasn't the women...

If matters unfold in such a way that FontForge will have been forked three times and all of them had the developers who left talking about "stability" and "we disagree with the development priorities of mainline FontForge," maybe it's time to ask some new questions.

**Comment by lpsandaruwan** [comment]
@mskala Someone has already tried to build a GTK UI, using glade's version 2. I don't know why he stopped it. I'm not sure about QT thing. Because I is not supporting C that much.

**Comment by larsenwork** [comment]
@mskala documention: I'd suggest we move all the old documentation to fontforge.github.io/old and hide it from robots as soon as we release the rearranged menu and updated documentation. I've talked with @davelab6 and @pathumego about finding a week were we can do a major push in updating documentation + finishing the menu rearrangement. 

I'm new to FontForge development so I don't understand what you're trying to say with you wife analogy. 

I'll create another issue where we can discuss name change - I think it's important especially if SourceForge hijacking is a valid concern (I know nothing about this).

**Comment by jtanx** [comment]
@lpsandaruwan the GTKUI attempt you see was by George Williams which never got completed.

**Comment by lpsandaruwan** [comment]
It is nice and smooth actually. Have no idea why he stopped it.
![gtkui](https://cloud.githubusercontent.com/assets/5220773/8143789/78fa5388-11db-11e5-8e2e-9773182f5d48.png)

**Comment by mskala** [comment]
Switching to a modern UI toolkit costs more than any single UI task, but the cost of not doing so must be paid on every single UI task.

**Comment by pathumego** [comment]
I missed out a lot. But I have a few things to add in about UI improvements. 

Fundamentally im just a user of FF. Haven't contributed a line of code :).  Im using FontForge daily and I have 4 other people collaborating with me, working full time on fonts at the moment. All together, 2 Mac users, 2 win users and 1 GNU/Linux. Most of the issues I've posted were from feedback and issues that we discussed. I agree totally with @frank-trampe about not doing things that would break what we have at the moment. So whatever we do in terms of UI improvements should be inside that frame. That is why Ive been cross posting some feature suggestions and improvements in here on FontForge and GlyphrStudio as well. So Glyphr could implement my wishes someday.

Anyway the menu improvements by @lpsandaruwan is happening in a branch, so I don't see any need for a full fork of the project. And Im sure @larsenwork and I can document the changes properly so there will be no issues or backfire from users. If someone wants the old menus we should provide support,documentation. 

Anyway I think `development` of a software is about all the things that we are talking about Stability, Bug Fixes, Performance improvements, UI Improvements and all of this. So I think we are lucky to have people who focus on all these different areas :) As a user I would love to see a software with all these aspects. So if we can figure out ways to implement the acceptable UI improvements @larsenwork and myself or anyone else suggests without breaking things, i don't see a problem or need to fork. If its not braking things and if its improving the software.
  
The naming issue, I think we just need a sane versioning system. I don't have enough knowledge on the packaging stuff.

**Comment by jtanx** [comment]
@frank-trampe with regards to the source tarball - you should be able to just alter [this line](https://github.com/fontforge/fontforge/blob/414578478f89c8ab6d62da96381b6b7cf5e1343a/configure.ac#L37) to the current package release date (perhaps there is a way to automatically do this in m4, I don't know), and if you run

    make dist

The resulting tarball should be usable. Since it copies everything when making the tarball, there are no symlinks and it automatically discards the git version history and gnulib folder.

**Comment by davelab6** [comment]
Matthew, I think the divorce analogy is a good one, but what I'm wanting is
advice on how to meet the needs of conservative development like frank has
proposed and your fontanvil (fork number 2, focused on native scripting
stability, right?) with radical engine development (fork number 1,
sortsmill) and with centrist ui improvement (post 2012 main line)

Sortsmill was explicitly forked to avoid discussion and community
involvement, and Khaled is now in a situation where his peers in Egypt can
not use his tools because they aren't intended for anyone other than those
2 developers. I consider this unsolvable, but i would be very happy if
Khaled joined us.

FontAnvil was forked for stability improvement and native scripting and up
to date documentation, right? I have some hope that you and frank could
collaborate on making main line fontforge meet those needs, but it's not
clear to be how far they have diverged already.

Personally I think the current ff ui effort has a few low hanging fruit
that are worth while but then focus should move to a web based app, and
Glyphr-Studio seems like the best bet. But I'm funding font development
only this year, and it's up to designers like Pathum and Andreas where they
spend their energy.

**Comment by KrasnayaPloshchad** [comment]
In this release, the original title was disappeared in the title bar, displaying "VcXsrv X" instead.
![no title](https://cloud.githubusercontent.com/assets/10364351/8479770/69eb837c-210c-11e5-941b-1ba5b72019f9.png)

**Comment by jtanx** [comment]
@KrasnayaPloshchad  when did you install it? If it was close to when it was first released, can you try re-downloading and re-installing it? I updated the binary at one point (but retained the same name) to fix this issue.

If you just downloaded it, then do you have any non-English/non-ASCII characters in your install path? Also, what version of Windows are you running (e.g. Windows 8.1 English Australia)

**Comment by KrasnayaPloshchad** [comment]
@jtanx Hi, I have just download and installed this release from your official website yesterday, I only installed in the default path, as your installer specified, no any non-English/non-ASCII characters in my install path.

Windows 8.1 China Edition, Yoga 13

**Comment by KrasnayaPloshchad** [comment]
I download again just now, but this problem still appearing to me.

**Comment by jtanx** [comment]
Interesting. @KrasnayaPloshchad, could you try running this [file](https://github.com/jtanx/fontforgebuilds/releases/download/v0.01/ShowLocale.exe) and tell me what it says? ( [source](https://github.com/jtanx/fontforgebuilds/blob/master/patches/ShowLocale/ShowLocale/showlocale.c) )

You should see something like this:
![showlocale](https://cloud.githubusercontent.com/assets/5137410/8499424/6e1cc046-21c5-11e5-91cd-8b8b7983e712.png)

Thanks!

**Comment by KrasnayaPloshchad** [comment]
@jtanx Hi, my result is here:

![no title](https://cloud.githubusercontent.com/assets/10364351/8502437/77fa50b6-21e2-11e5-9973-0bb5a1c35b93.png)

**Comment by jtanx** [comment]
@KrasnayaPloshchad, thanks for that! If you add:

~~~~~
Chinese_(Simplified)_China.936:	zh_CN.EUC
~~~~~

To the end of the file `C:\Program Files (x86)\FontForgeBuilds\FontForge\bin\VcXsrv\locale\locale.alias` I think it should fix your problem. I'll see if I can make an updated build in the near future.

**Comment by jtanx** [comment]
@KrasnayaPloshchad can you try [this build](https://github.com/fontforge/fontforge/releases/download/20150612/FontForge-2015-06-12-Windows2.exe)?

**Comment by KrasnayaPloshchad** [comment]
@jtanx Well, the problem is still there.

**Comment by jtanx** [comment]
@KrasnayaPloshchad ugh the original settings (that came with X11) were wrong. You need `zh_CN.eucCN` instead of `zh_CN.EUC`. I'll upload another build again soon...

**Comment by KrasnayaPloshchad** [comment]
@jtanx OK, the title is displaying properly.

**Comment by davelab6** [comment]
The June release was made and the following discussion is over, so closing
